build.sbt

```diff
@@ -18,28 +18,34 @@ lazy val supportedScalaVersions = List(scala212, scala211)
 val silencerV_2_12 = "1.6.0"
 val silencerV = "1.7.0"
 
+def propOrEnv(name: String, default: String = null) = Option(System.getProperty(name)).orElse(sys.env.get(name)).getOrElse(default)
+def propOrEnv(name: String) = Option(System.getProperty(name)).orElse(sys.env.get(name))
```

Comment:
Hmm.. How does it work? For example: it looks for `includeFlinkAndScala` in evns? I think it's ugly..  I mean when we run ` propOrEnv("includeFlinkAndScala")` we should look for property `includeFlinkAndScala` or env `INCLUDE_FLINK_AND_SCALA`. But I didn't look to method  `sys.env.get`

Reply:
I added TODO to get rid of properties and switch to ENV